box.json

```diff
@@ -0,0 +1,15 @@
+{
+    "alias": "roave-backward-compatibility-check.phar",
```

Comment:
is the alias really needed?

Reply:
Maybe not? I copied it from Doctrine Migrations.